crates/billing-integrations/src/stripe.rs

```diff
@@ -243,47 +256,42 @@ impl Invoice {
             Some(inv) => inv,
             None => {
                 let invoice = stripe::Invoice::create(
-            client,
-            stripe::CreateInvoice {
-                customer: Some(customer.id.to_owned()),
-                // Stripe timestamps are measured in _seconds_ since epoch
-                // Due date must be in the future
-                due_date: if date_end_secs > timestamp_now { Some(date_end_secs) } else {Some(timestamp_now + 10)},
-                description: Some(
-                    format!(
-                        "Your Flow bill for the billing preiod between {date_start_human} - {date_end_human}"
-                    )
-                    .as_str(),
-                ),
-                collection_method: Some(stripe::CollectionMethod::SendInvoice),
-                auto_advance: Some(false),
-                custom_fields: Some(vec![
-                    stripe::CreateInvoiceCustomFields {
-                        name: "Billing Period Start".to_string(),
-                        value: date_start_human.to_owned(),
+                    client,
+                    stripe::CreateInvoice {
+                        customer: Some(customer.id.to_owned()),
+                        // Stripe timestamps are measured in _seconds_ since epoch
+                        // Due date must be in the future. Bill net-30, so 30 days from today
+                        due_date: Some((Utc::now() + Duration::days(30)).timestamp()),
+                        description: Some(
+                            format!(
+                                "Your Flow bill for the billing preiod between {date_start_human} - {date_end_human}. Tenant: {tenant}",
```

Comment:
nit: preiod

Reply:
Period at the end after the tenant name? Or you don't want the period after the billing period and tenant?

Reply:
Lol I was point out that period is misspelled.

Reply:
d'oh! 🤦